functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
For the schedule smo there is no distinction between a schedule and a shared schedule. I can easily create a schedule and add it to another job based on the ID. 

The code below will use two existing jobs called "Job1" an "Job2" to add a schedule to. 
```
Import-Module SqlServer

# Create the server object
$server = New-Object Microsoft.SqlServer.Management.Smo.Server '.'

# Set some names
$jobName1 = 'Job1'
$jobName2 = 'Job2'
$scheduleName = 'TestSharedSchedule'

# Get the jobs
$smoJob1 = $Server.JobServer.Jobs[$jobName1]
$smoJob2 = $Server.JobServer.Jobs[$jobName2]

# Get the count for the schedules
$smoJob1.JobSchedules.Count
$smoJob2.JobSchedules.Count

# Create the job schedule
$smoSchedule = New-Object Microsoft.SqlServer.Management.Smo.Agent.JobSchedule($smoJob1, $scheduleName)

# Set some values for the schedule
$smoSchedule.ActiveStartDate = Get-Date
$smoSchedule.ActiveEndDate = '9999-12-31'
$smoSchedule.FrequencyTypes = "Daily"
$smoSchedule.FrequencyInterval = 1
$smoSchedule.ActiveStartTimeOfDay = 230000
$smoSchedule.ActiveEndTimeOfDay = 235959

# Create the schedule for the job
$smoSchedule.Create()

$smoJob2.AddSharedSchedule($smoSchedule.ID)

Write-Host "`nNormal schedules`n" -ForegroundColor Cyan
$($server.JobServer.Jobs.JobSchedules.Name)

Write-Host "`nShared Schedules`n" -ForegroundColor Cyan
$($server.JobServer.SharedSchedules.Name)
```

At first the jobs have no schedule:
![image](https://cloud.githubusercontent.com/assets/6154981/26829220/f187d36a-4ac4-11e7-9d05-466cf8543beb.png)

After the script:
![image](https://cloud.githubusercontent.com/assets/6154981/26829257/1dabbc36-4ac5-11e7-8ab6-1d187e957f6d.png)

What we might want to do is add a feature to the set-dbaagentjob function to add a shared schedule based on the ID or the name of the schedule. 

I don't think the other functions need to be changed.